### DISCUSSION TITLE
Flax to ONNX?

### DISCUSSION THREAD

**Discussion by GiuseppeDiGuglielmo** [google/flax#3656] [Q&A]
Can a Flax model be converted to [ONNX](https://onnx.ai)?

**Comment by chiamp** [comment]
I'm not familiar with ONNX, but I found a [Python notebook](https://gist.github.com/maltempi/f2d30e513ef7046f3c0455a5dc4be3e0) that seems to load ONNX params into a Flax model. Perhaps the other way around is possible?
